modules/hcp-eks-client/templates/consul.tpl

```diff
@@ -28,17 +37,16 @@ externalServers:
 server:
   enabled: false
 
-client:
-  enabled: true
-  join: ${consul_hosts}
-  nodeMeta:
-    terraform-module: "hcp-eks-client"
-
 connectInject:
   transparentProxy:
-    defaultEnabled: true
+    defaultEnabled: false
   enabled: true
   default: true
+%{ if !consul_client_agent ~}
+  consulNode:
+    meta: "hcp-eks-agentless-node"
+%{ endif ~}
+
```

Comment:
validate this before GA: required for HCP analytics (currently nothing happens if this is included as consul-k8s does not support it yet)